apps/site/src/components/pages/hub/block-data-container.tsx

```diff
@@ -197,7 +197,7 @@ export const BlockDataContainer: FunctionComponent<BlockDataContainerProps> = ({
           }}
         >
           <Tab label="Preview" />
-          <Tab label="Source Code" />
+          <Tab label="Block Data" />
```

Comment:
Drive-by: this is the name for the tab in mobile which contains the block example properties and block schema. I think 'Source Code' is an inappropriate name for these things so I've renamed it.